apps/src/studio/paramLists.js

```diff
@@ -22,6 +22,15 @@ exports.getPlaySoundValues = function (withRandom) {
     names = [];
   }
   names = names.concat(skin.sounds);
+  if (withRandom) {
+    // Insert a random value for each sound group before the first sound in the group:
+    for (var group in skin.soundGroups) {
+      var insertIndex = names.indexOf(group + skin.soundGroups[group].minSuffix);
+      if (insertIndex != -1) {
+        names.splice(insertIndex, 0, skin.soundGroups[group].randomValue);
```

Comment:
*gasp* `for-in` without `hasOwnProperty`?

:grin: No action item, that warning is actually a pet peeve.  This is cool.